# <mayhem_program::instructions::fee_recipients::FeeRecipient as anchor_lang::AccountSerialize>::try_serialize  (yours 328 B at .text+0x30fe8; original 544 B at .text+0x3ca38)
# distance 45 instructions; evidence: called-by <mayhem_program::instructions::fee_recipients::InitializeFeeRecipient as anchor_lang::AccountsExit>::exit#0174; same-callees 2/2; neighbours between the same matched functions as in your build
--- original
+++ yours
@@ -1,67 +1,40 @@
-mov64 r6, r2
-mov64 r7, r1
-mov64 r1, r6
+mov64 r7, r2
+mov64 r6, r1
+mov64 r1, r7
 lddw r2, .rodata[94fad18ab9eac3e3]
 mov64 r3, 8
 call <anchor_lang::bpf_writer::BpfWriter<&mut [u8]> as std::io::Write>::write_all#7274
-jne r0, 0, +0x1a0
-mov64 r2, r7
+jne r0, 0, +0xc8
+mov64 r2, r6
 add64 r2, 32
-mov64 r1, r6
+mov64 r1, r7
 mov64 r3, 1
 call <anchor_lang::bpf_writer::BpfWriter<&mut [u8]> as std::io::Write>::write_all#7274
-jne r0, 0, +0x1a0
-mov64 r2, r7
+jne r0, 0, +0xc8
+mov64 r2, r6
 add64 r2, 33
-mov64 r1, r6
+mov64 r1, r7
 mov64 r3, 1
 call <anchor_lang::bpf_writer::BpfWriter<&mut [u8]> as std::io::Write>::write_all#7274
-jne r0, 0, +0x1a0
-ldxdw r1, [r7]
-stxdw [r10-8], r1
-mov64 r2, r10
-add64 r2, -8
-mov64 r1, r6
-mov64 r3, 8
+jne r0, 0, +0xc8
+mov64 r1, r7
+mov64 r2, r6
+mov64 r3, 32
 call <anchor_lang::bpf_writer::BpfWriter<&mut [u8]> as std::io::Write>::write_all#7274
-jne r0, 0, +0x1a0
-ldxdw r1, [r7+8]
-stxdw [r10-8], r1
-mov64 r2, r10
-add64 r2, -8
-mov64 r1, r6
-mov64 r3, 8
-call <anchor_lang::bpf_writer::BpfWriter<&mut [u8]> as std::io::Write>::write_all#7274
-jne r0, 0, +0x1a0
-ldxdw r1, [r7+16]
-stxdw [r10-8], r1
-mov64 r2, r10
-add64 r2, -8
-mov64 r1, r6
-mov64 r3, 8
-call <anchor_lang::bpf_writer::BpfWriter<&mut [u8]> as std::io::Write>::write_all#7274
-jne r0, 0, +0x1a0
-ldxdw r1, [r7+24]
-stxdw [r10-8], r1
-mov64 r2, r10
-add64 r2, -8
-mov64 r1, r6
-mov64 r3, 8
-call <anchor_lang::bpf_writer::BpfWriter<&mut [u8]> as std::io::Write>::write_all#7274
-jeq r0, 0, +0x210
+jeq r0, 0, +0x138
 mov64 r1, r0
 and64 r1, 3
 mov64 r2, r1
 add64 r2, -2
-jlt r2, 2, +0x1f8
-jeq r1, 0, +0x1f8
+jlt r2, 2, +0x120
+jeq r1, 0, +0x120
 ldxdw r1, [r0+7]
 ldxdw r2, [r1]
-jeq r2, 0, +0x1f8
+jeq r2, 0, +0x120
 ldxdw r1, [r0-1]
 callx r2
 mov64 r1, 3004
 call <anchor_lang::error::Error as core::convert::From<anchor_lang::error::ErrorCode>>::from#ae9e
 exit
 mov64 r0, 2
-ja +0x208
+ja +0x130
